playbook/loadtest/main.yml

```diff
@@ -15,7 +15,7 @@
   vars:
     ansible_ssh_pipelining: no
     results_fn: "{{ lookup('pipe', 'date +%Y%m%d%H%M') }}_loadtest_results.log"
-    src_results_file: "/home/ec2-user/bluebutton-web-deployment/locust/{{ results_fn }}"
```

Comment:
the flexibility of this being a `var` seems good, why the change to hardcoding?

Reply:
Was having issues with the timestamps being different, since the date was being executed when the variable was being referenced. So was getting different dates where the var was being used. So just setting the timestamp on the file for the S3 copy.